go/ct/rlz/parameters.go

```diff
@@ -154,6 +154,9 @@ var gasParameterSamples = []U256{
 	NewU256(1),
 	NewU256(1 << 10),
 	NewU256(st.MaxGasUsedByCt),
+	NewU256(math.MaxInt64),
+	NewU256(math.MaxInt64 + 1),
+	NewU256(math.MaxUint64),
```

Comment:
it is a parameter type that is only used in the call rules. Which is where we  want to stress it, not sure we can avoid using them.

Reply:
The last value can be removed (maxuint64) since maxint+1 already checks numbers larger than boundary. 
- 1<<10 is an arbitrary value, could be removed
- MaxGasUsedByCt is no longer the max gas used by the CT, we need to reconsider this constant, where does it come from and what does it mean now

Reply:
discussed offline, resolved: delete `1<<10` and `st.MaxGasUsedByCt` since it is from another domain. I will leave however `NewU256(math.MaxInt64)` and `NewU256(math.MaxInt64 + 1)`.